templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -15,13 +15,13 @@ contents:
     # find default interface
     while [ $counter -lt 12 ]; do
       # check ipv4
-      iface=$(ip -j route show default | jq -r '.[0].dev')
+      iface=$(ip route show default | awk '{if ($4 == "dev") print $5; exit}')
       if [[ -n "$iface"  && "$iface" != "null" ]]; then
         echo "IPv4 Default gateway interface found: ${iface}"
         break
       fi
       # check ipv6
-      iface=$(ip -6 -j route show default | jq -r '.[0].dev')
+      iface=$(ip -6 route show default | awk '{if ($4 == "dev") print $5; exit}')
       if [[ -n "$iface"  && "$iface" != "null" ]]; then
```

Comment:
dont need the null check anymore since we remove jq

Reply:
done